vm/mikrotik-routeros.sh

```diff
@@ -62,7 +62,7 @@ function cleanup() {
 }
 TEMP_DIR=$(mktemp -d)
 pushd $TEMP_DIR >/dev/null
- if ! pveversion | grep -Eq "pve-manager/(7.4-[1][3-9]|8.1.[1-9])"; then
```

Comment:
I did the simplest way I found because I do not know the reason why the 8.1.x was specified while the message tells "or later"